.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
This doesn't belong to the repo, it's probably just some local config for you. Don't commit it.